internal/handlers/common/tail.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"github.com/FerretDB/FerretDB/internal/pg"
+	"github.com/FerretDB/FerretDB/internal/types"
+)
+
+// Tail creates a tail for sql find and count queries.
+func Tail(castString string, m map[string]any, placeholder *pg.Placeholder) (sql string, args []any, err error) {
+	sort, _ := m["sort"].(*types.Document)
```

Comment:
it was just a refactoring there, I prefer to left as it is in this request.

Reply:
sort and limit as parameters will increase parameters count to more that 4, it's not very convenient... WDYT?

Reply:
Hm, I think it would be 4 parameters as we won't need `m map[string]any`:
```go
func Tail(castString string, sort *types.Document, limit int32, placeholder *pg.Placeholder) (sql string, args []any, err error)
```

Reply:
applied.